packages/portalnetwork/src/subprotocols/history/headerAccumulator.ts

```diff
@@ -103,26 +103,25 @@ export class AccumulatorManager {
     this.init()
   }
 
-  public get header() {
-    return this.headerAccumulator
-  }
-
   public updateAccumulator(newHeader: BlockHeader) {
     this.headerAccumulator.updateAccumulator(newHeader)
   }
 
-  public get currentHeight() {
+  public currentHeight() {
```

Comment:
:+1: 

I prefer this approach, anyway.  Whoever built the headerAccumulator must have left some of their `get`s around...

Reply:
Probably, never can trust that guy.